srcpkgs/libdispatch/template

```diff
@@ -0,0 +1,32 @@
+pkgname=libdispatch
+version=5.5.3
+revision=1
+create_wrksrc=yes
+build_style=cmake
+hostmakedepends="ninja clang git"
+short_desc="Library provides comprehensive support for concurrent code execution on multicore hardware"
+maintainer="Ilya <ISPonomarev@yahoo.com>"
+license="Apache-2.0"
+homepage="https://github.com/apple/swift-corelibs-libdispatch"
+
+do_fetch() {
+       git clone https://github.com/apple/swift-corelibs-libdispatch
```

Comment:
We only accept releases, not building from the master branch.

Reply:
Build from release failed, from master successful

Reply:
If there are patches that are needed, they can be included, but it would be preferred that there be a new, working release.